index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <script
+      src="https://kit.fontawesome.com/198fcc7e17.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <main>
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name"><a href="#">Kevin Toro</a></p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
+        <section class="headline-section">
```

Comment:
- [ ] Please as requested by the previous code reviewer, the `header` tag should only contain the logo, the menu icon, and the nav link only. Also, make sure the header should not include in the main tag for semantic reasons.